manifest.json

```diff
@@ -1,28 +1,52 @@
 {
-  "name": "@deskpro-apps/my-app",
-  "title": "My App",
-  "description": "My Deskpro app",
+  "name": "@deskpro-apps/woocommerce",
+  "title": "WooCommerce",
+  "description": "Take control of your orders with the WooCommerce app",
   "version": "0.0.1",
   "scope": "agent",
   "isSingleInstall": false,
   "hasDevMode": true,
-  "targets": [{ "target": "ticket_sidebar", "entrypoint": "index.html" }],
+  "targets": [
+    {
+      "target": "user_sidebar",
+      "entrypoint": "index.html"
+    }
+  ],
   "settings": {
-    "example": {
-      "title": "Example Setting",
-      "description": "Example backend only setting",
+    "consumer_key": {
+      "title": "Consumer Key",
+      "description": "Consumer key can be obtained by following our setup guide",
+      "type": "string",
+      "isRequired": true,
+      "isBackendOnly": true,
+      "order": 10
+    },
+    "consumer_secret": {
+      "title": "Consumer Secret",
+      "description": "Consumer secret can be obtained by following our setup guide",
+      "type": "string",
+      "isRequired": true,
+      "isBackendOnly": true,
+      "order": 20
+    },
+    "store_url": {
```

Comment:
Add validation url please for better UX.

example: https://github.com/DeskproApps/gitlab/blob/master/manifest.json#LL50C19-L50C19